### DISCUSSION TITLE
近期反馈整理表

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3333] [讨论]
由于近期龙猫鸽太久了没更新，我怕刚整理完龙猫就更新了。所以这个 Discussion 将在 2.6.14 发布之后正式启用。

**Comment by Deep-Dark-Forest** [comment]
笑死，我们没有作业（

**Reply by allMagicNB** [reply]
大学生是吧

**Reply by iPodtouch6557** [reply]
快乐教育（）

**Comment by Ad-closeNN** [comment]
~~打算≠一定做~~

**Comment by iPodtouch6557** [comment]
6，占坑是吧

**Reply by Deep-Dark-Forest** [reply]
哈哈哈#3333这个靓号让他给占了

**Reply by hejiehao** [reply]
我怀疑 #3333 这个数字就是看准时机开的

**Reply by allMagicNB** [reply]
![Image_1708157437013.png](https://github.com/Hex-Dragon/PCL2/assets/116283787/296b2f56-d74e-44da-aa31-3cdfc34ead31)

**Reply by iPodtouch6557** [reply]
6
话说github有五位数的issue号吗

**Reply by hejiehao** [reply]
什么时候 PCL2 能攒到 6666

**Reply by Deep-Dark-Forest** [reply]
> 6 话说github有五位数的issue号吗

![image](https://github.com/Hex-Dragon/PCL2/assets/131975043/4099c378-6b24-449d-9b9c-509dbe39e388)
6位数都有好吧

**Reply by Ad-closeNN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/58c5cf8c-9862-4609-90e9-ad4fa3d66364)
6pr也在稳定发挥

**Reply by zkitefly** [reply]
> ![Image_1708157437013.png](https://github.com/Hex-Dragon/PCL2/assets/116283787/296b2f56-d74e-44da-aa31-3cdfc34ead31)

还有这种群

**Reply by allMagicNB** [reply]
> 还有这种群

@Pigeon0v0 建的，但龙猫似乎不倡导……
现在一共有 6 位协作者和 2 位特别活跃的志愿者（
群号是379026872

**Comment by Ad-closeNN** [comment]
> 6 话说github有五位数的issue号吗

~~开个测试repo，用bot给issue一直刷反馈~~

**Comment by wuliaodexiaoluo** [comment]
> > 6 话说github有五位数的issue号吗
> 
> ~开个测试repo，用bot给issue一直刷反馈~

相信龙猫会送出封号大礼包

**Reply by Ad-closeNN** [reply]
不是龙猫，是GitHub官方，因为是自开repo
